src/seekcentral.cpp

```diff
@@ -423,28 +434,31 @@ namespace Sleipnir {
             }
 
             if (isFirst) {
-                string err = "Error: no dataset contains any of the query genes";
-                fprintf(stderr, "%s\n", err.c_str());
+                string err = "Error: no dataset contains enough query genes";
+                fprintf(stderr, "%s (%d)\n", err.c_str(), l);
                 if (m_bEnableNetwork) {
                     CSeekNetwork::Send(m_iClient, err);
                 } else if (m_useRPC) {
                     throw query_error(FILELINE + err);
                 }
-                return false;
+                if (m_vecstrAllQuery.size() == 1)
```

Comment:
Should we add a test case where there are multiple queries in the input file, but at least one query has no genes in the datasets? To test the continue with the other queries functionality.

Reply:
Ah, yeah that's a good idea

Reply:
There is already a SeekMiner test that runs a list of queries from a file. We could add a query to that list that would be excluded by checkDatasets.